src/main.ts

```diff
@@ -33,16 +51,42 @@ function makeWriter(): CodeWriter {
   };
 }
 
-export function generateRuntypes(...roots: RootType[]): string {
+interface GenerateOptions {
```

Comment:
```suggestion
export interface GenerateOptions {
```

In case anyone ever needs it 🤷‍♂️